src/components/Namestage.tsx

```diff
@@ -129,7 +112,7 @@ function Namestage({
         error={toggleError}
         onChange={handleValidation}
```

Comment:
error props seems to take a function but the name suggest it expecting a object or value! use better name here for props